test/addons/async-hello-world/binding.cc

```diff
@@ -13,7 +20,8 @@ struct async_req {
 
 void DoAsync(uv_work_t* r) {
   async_req* req = reinterpret_cast<async_req*>(r->data);
-  sleep(1);  // Simulate CPU intensive process...
+  // Simulate CPU intensive process...
+  std::this_thread::sleep_for(std::chrono::seconds(1));
```

Comment:
I noticed the default libc++ library on OS X is outdated and doesn't support c++11. I wonder how OS X user compile the latest v8, which is part of nodejs and using c++11? they should have a solution to that. (I think they download the lib from somewhere)